.Gruntfile.babel.js

```diff
@@ -227,7 +234,7 @@ function browserifyCfg ({straight, lazy}, cfg = {}) {
   function gencfg (out, paths, isLazy) {
     var c = {
       options: {
-        transform: [script2ify, 'vueify', ejsify, 'babelify'],
+        transform: [localify, script2ify, 'vueify', ejsify, 'babelify'],
```

Comment:
Yeah, but the translation strings only need to be updated once per release and sent to the localizers, so that can be done in a `grunt genstrings` command or something.